specification/ai/Azure.AI.Unified/evaluations/models.tsp

```diff
@@ -0,0 +1,90 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@azure-tools/typespec-autorest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "../common/models.tsp";
+import "../main.tsp";
+import "@typespec/openapi";
+
+using TypeSpec.OpenAPI;
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+
+namespace Azure.AI.Unified;
+
+@doc("Evaluator Configuration")
+model EvaluatorConfiguration {
+  @doc("Identifier of the evaluator.")
+  id: string;
+
+  @doc("Initialization parameters of the evaluator.")
+  initParams?: Record<unknown>;
+
+  @doc("Data parameters of the evaluator.")
+  dataMapping?: Record<string>;
+}
+
+#suppress "@azure-tools/typespec-azure-core/no-string-discriminator"
+@doc("Abstract data class.")
+@discriminator("type")
+model InputData {
+  @doc("Type of the data")
+  type: string;
```

Comment:
What kinds of `InputData` do we expect to have? Right now, there is only one. But as indicated by the suppressions on line 31, we should create a union for the discriminator.